mathics_scanner/data/named-characters.yml

```diff
@@ -6919,6 +6930,12 @@ Upsilon:
   unicode-equivalent-name: GREEK SMALL LETTER UPSILON
   wl-unicode: "\u03C5"
   wl-unicode-name: GREEK SMALL LETTER UPSILON
+
+# The rendering of Uranus at https://reference.wolfram.com/language/ref/character/Uranus.html
+# looks more like U+26E2 (Astronomical Symbol for Uranus) than the Standard Unicode equavalent
+# seen at https://www.compart.com/en/unicode/U+2645.
+# As with the Earth, we are going off of the name and the code point rather than the
+# visual representation of the symbo.
```

Comment:
There's a little typo in here.

Reply:
Fixed on https://github.com/Mathics3/mathics-scanner/pull/1/commits/659749078e08aafa80bec8e727bf65fae91efed4